temperature/models.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import Column, Integer, ForeignKey, DateTime, Float
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class Temperature(Base):
+    __tablename__ = "temperature"
+
+    id = Column(Integer, primary_key=True, index=True)
+    city_id = Column(Integer, ForeignKey("city.id"), nullable=False)
+    date_time = Column(DateTime, nullable=False)
+    temperature = Column(Float, nullable=False)
+
+    city = relationship("City")
```

Comment:
The `city` relationship is defined to link the `Temperature` model to the `City` model. Ensure that the `City` model is correctly defined and accessible in your project.